README.md

```diff
@@ -1,44 +1,58 @@
 # sphinxcontrib-drawio
-Sphinx Extension to add the ``drawio`` directive to include draw.io diagrams.
-
-**Important:** This extension does not work on readthedocs as RTD does not allow packages (e.g. drawio) to be installed.
-It would be better to use editable SVGs or PNGs, accessible through draw.io's File > Export menu.
+Sphinx Extension adding the `drawio-image` and `drawio-figure` directives.
+These are equivalent to the standard `image` and `figure` directives, but
+accept the path to a `.drawio` file and additional options to control
+exporting of the diagram to a format suitable for the selected Sphinx builder. 
+
+These directives replace the `drawio` directive which is now deprecated, and
```

Comment:
Please add a note that these will be removed in version 1.0 [I think this makes sense?]

Reply:
Done.